platform/src/main/python/dlpx/virtualization/platform/_linked.py

```diff
@@ -272,8 +334,8 @@ def _internal_staged_pre_snapshot(self, request):
         staged_mount = request.staged_source.staged_mount
         mount = Mount(remote_environment=RemoteEnvironment.from_proto(
             staged_mount.remote_environment),
-                      mount_path=staged_mount.mount_path,
-                      shared_path=staged_mount.shared_path)
+            mount_path=staged_mount.mount_path,
+            shared_path=staged_mount.shared_path)
```

Comment:
This changes seems like removing whitespaces? Is it because of some linting issue?